examples/shared_vpc/main.tf

```diff
@@ -42,13 +42,15 @@ provider "random" {
   Host Project Creation
  *****************************************/
 module "host-project" {
-  source               = "../../"
-  random_project_id    = true
-  name                 = var.host_project_name
-  org_id               = var.organization_id
-  folder_id            = var.folder_id
-  billing_account      = var.billing_account
-  skip_gcloud_download = true
+  source                         = "../../"
+  random_project_id              = true
+  name                           = var.host_project_name
+  org_id                         = var.organization_id
+  folder_id                      = var.folder_id
+  billing_account                = var.billing_account
+  skip_gcloud_download           = true
+  shared_vpc                     = var.host_project_name
```

Comment:
We shouldn't need this on the host project.

Reply:
again, shared vpc is the host project name... it's better to repeat here than add an if in the code